### ISSUE TITLE
pkg-config: macports shouldn't take priority over custom paths in OSXCROSS_PKG_CONFIG_PATH

### ISSUE THREAD

**Issue by Akaricchi** [tpoechtrager/osxcross#132]
Having the macports entry on the very top makes it unnecessarily difficult to override installed macports packages with custom built ones.

**Comment by tpoechtrager** [comment]
Good point. Could you file a pull request? The source code for the tool is in `wrapper/program/pkg-config.cpp`.

**Comment by Akaricchi** [comment]
I'll try to find some time soon.

**Comment by antmak** [comment]
Have you any changes?
